### PR TITLE
Remove forced background paint

### PR DESCRIPTION
**Pull request by MikhailArkhipov**
And simplify main thread paint a bit

### CODE REVIEW

File: src/Package/Impl/DataInspect/VisualGrid/VisualGridScroller.cs [hunk 1]
```diff
@@ -197,22 +195,19 @@ internal sealed class VisualGridScroller {
                     throw new InvalidOperationException("Couldn't acquire enough data");
                 }
 
-                // actual drawing runs in UI thread
-                await Task.Factory.StartNew(
-                    () => DrawVisuals(newViewport, data, updateType, visualViewport, suppressNotification),
-                    token,
-                    TaskCreationOptions.None,
-                    _ui);
+                await DrawVisualsAsync(newViewport, data, updateType, visualViewport, suppressNotification);
```
Comment (AlexanderSher) [inline src/Package/Impl/DataInspect/VisualGrid/VisualGridScroller.cs]: Need to pass `token` to the `SwitchToMainThreadAsync`

Comment (MikhailArkhipov) [inline src/Package/Impl/DataInspect/VisualGrid/VisualGridScroller.cs]: Sure

**Comment by MikhailArkhipov** [comment]
This change in attempt to fix grid being blank while data is actually there.

**Comment by AlexanderSher** [comment]
👍

**Pull request merged**